include/seqan3/range/views/enforce_random_access.hpp

```diff
@@ -6,365 +6,13 @@
- * | std::ranges::bidirectional_range |                                   | *guaranteed*                               |
- * | std::ranges::random_access_range |                                   | *guaranteed*                               |
- * | std::ranges::contiguous_range    |                                   | *preserved*                                |
- * |                                  |                                   |                                            |
- * | std::ranges::viewable_range      | *required*                        | *guaranteed*                               |
- * | std::ranges::view                |                                   | *guaranteed*                               |
- * | std::ranges::sized_range         |                                   | *preserved*                                |
- * | std::ranges::common_range        |                                   | *preserved*                                |
- * | std::ranges::output_range        |                                   | *preserved*                                |
- * | seqan3::const_iterable_range     |                                   | *preserved*                                |
- * |                                  |                                   |                                            |
- * | std::ranges::range_reference_t   |                                   | std::ranges::range_reference_t<urng_t>     |
- *
- * See the \link views views submodule documentation \endlink for detailed descriptions of the view properties.
- *
- * This adaptor requires that the underlying range models either std::ranges::random_access_range or
- * seqan3::pseudo_random_access_range.
- *
- * ### Return type
- *
- * | `urng_t` (underlying range type)       | `rrng_t` (returned range type)                       |
- * |:--------------------------------------:|:----------------------------------------------------:|
- * | `std::ranges::random_access_range`     | `std::ranges::ref_view<urng_t>`                      |
- * | `seqan3::pseudo_random_access_range`   | `seqan3::detail::view_enforce_random_access`         |
- *
- * The adaptor returns exactly the type specified above. In the second case a view is returned whose iterator wraps
- * the iterator of the underlying range and adapts all of its functionality but overwrites the
- * iterator category to be std::random_access_iterator_tag.
- *
- * ### Example
- *
- * \include test/snippet/range/views/enforce_random_access.cpp
- *
- * ### Complexity
- *
- * Construction of the returned view is in \f$ O(1) \f$.
- *
- * \hideinitializer
- */
-inline constexpr auto enforce_random_access = detail::pseudo_random_access_fn{};
-//!\}
-} // namespace seqan3::views
+SEQAN3_DEPRECATED_HEADER("This header is deprecated and will be removed in SeqAn-3.1.0; Please #include <seqan3/utility/views/enforce_random_access.hpp> instead.")
```

Comment:
This could be made shorter, if wanted too.
```suggestion
SEQAN3_DEPRECATED_HEADER(
    "This header is deprecated and will be removed in SeqAn-3.1.0; Please #include <seqan3/utility/views/enforce_random_access.hpp> instead.")
```